packages/client/src/App.tsx

```diff
@@ -1,20 +1,30 @@
 import { ThemeProvider } from 'styled-components'
 import { useChangeTheme } from './hooks/useChangeTheme'
 import { GlobalStyles } from './assets/styles/globalStyle'
-import { RouterProvider } from 'react-router-dom'
-import { router } from './router'
+import { BrowserRouter } from 'react-router-dom'
+import { Router } from './router'
 import PageWrapper from './pages/PageWrapper'
+import { useEffect } from 'react'
+import { useAppDispatch } from './hooks/store'
+import { setIsAuth } from './store/auth/AuthSlice'
 
 function App() {
-  const { theme, themeToggler } = useChangeTheme()
+  const { theme } = useChangeTheme()
+  const dispatch = useAppDispatch()
+
+  useEffect(() => {
+    const isAuth = localStorage.getItem('isAuth')
+    dispatch(setIsAuth(isAuth === 'true'))
```

Comment:
Обычно то взаимодействие с localStorage делают через `JSON.stringify`, `JSON.parse`. Например
```suggestion
    const isAuth = JSON.parse(localStorage.getItem('isAuth') || 'false')
    dispatch(setIsAuth(isAuth))
```